editor/utils/test/url.js

```diff
@@ -1,25 +1,20 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { addQueryArgs } from '../url';
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.